src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
+    
+    If Not changeFileExtension(ThisWorkbook.Path, ThisWorkbook.Name) Then
+            
+    End If
+        
+    If Not windowsUnZip(Left(ThisWorkbook.FullName, Len(ThisWorkbook.FullName) - 4) & "zip", ThisWorkbook.Path) Then
```

Comment:
Assuming that the file extension is 4 characters long seems like a dangerous assumption. Perhaps FSO.GetBaseName or something similar would be more robust. Alternatively, the new name of the zip file is already evaluated in the changeFileExtension function, there is a chance here to just calculate it once and use it for both operations.